docs/src/changelog.md

```diff
@@ -261,7 +261,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - `Percival.jl` (#3567)
  - Added new tutorials:
     - [Approximating nonlinear functions](@ref) (#3563)
-    - [Classifiers](@ref) (#3569)
+    - [Example: classification problems](@ref) (#3569)
```

Comment:
Just checking you intended to rewrite the changelog?

Reply:
Yip

Reply:
Ah, it needs to match the new ref.